dts/bindings/iio/adc/microchip,mcp3204.yaml

```diff
@@ -0,0 +1,5 @@
+description: Microchip MCP3204
```

Comment:
I might have missed something, but don't we need copyright and SPDX boilerplate in bindings too ?

Reply:
YAML files are excluded in the GitHub license check workflow (see 3b625aaaa97b33d224e2c0445c441e481ccc4971 by @nashif).

Reply:
... and most of yaml bindings have Licences. Besides, I can't see why there would be a derogation specifically on these files.
So what should be the policy? @MaureenHelm , @nashif

Reply:
There are approximately 120 files under `dts/bindings/` without license and copyright (and approximately 400 files with license and copyright). I suggest leaving this PR as-is and opening a separate issue for dealing with the policy on licenses for YAML files.

Reply:
Thanks for this counting. Ok to deal with that later.